### PR TITLE
Added basic Tinder Social capabilities, start #73.

### PR DESCRIPTION
**Pull request by RNabel**
Add capability to

- Get list of facebook friends who are signed up to Tinder Social
- Create link to facebook profile of each of the friends
- Get Tinder profile information of each friend (i.e. information which is held by [models.User](https://github.com/charliewolf/pynder/blob/master/pynder/models/user.py#L8))

See `examples/tinder_social_examples.py` for an example.

### CODE REVIEW

File: pynder/models/friend.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import re
+from .user import User
+
+
+class Friend:
```
Comment (charliewolf) [inline pynder/models/friend.py]: please use new-style class (`class Friend(object)` instead of `class Friend`)

File: pynder/models/friend.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    facebook_id_pattern = re.compile(u'\.com/(\d+?)/')
```
Comment (charliewolf) [inline pynder/models/friend.py]: This is a constant so it shouldn't be a class attribute

**Comment by charliewolf** [comment]
This is awesome! I left a few minor notes but other than those this will be great to merge

**Comment by RNabel** [comment]
@charliewolf Thank you for the quick review. I made the changes!

**Pull request merged**